adr/0014-home-assistant-supervised.md

```diff
@@ -27,6 +27,7 @@ Docker CE (Community Edition) is the only supported containerization method for
 - NetworkManager >= 1.14.6
 - AppArmor == 2.13.x (built into the kernel)
 - Debian Linux Debian 10 aka Buster (no derivatives)
+- jq = 1.5
```

Comment:
It doesn't have to be 1.5 specifically, did you notice otherwise?

Besides, the Supervised doesn't need it, but the installer does. I suggest this is being added to the installer documentation instead.

Reply:
Unresolved this comment as the question wasn't answered.

Additionally: Why did you change it without a version in this case?

I still believe this is a better fit @ the installer documentation.

@mrdavidaylward ?